src/client/reducers/reducer_movies.js

```diff
@@ -0,0 +1,12 @@
+import C from '../actions/constants';
+
+export default function (state = null, action) {
```

Comment:
@vitkuz let's describe default state for each reducer

Reply:
@algbeta  done